url-list.csv

```diff
@@ -38,7 +38,7 @@ https://geth.ethereum.org/docs/rpc/ns-miner,https://geth.ethereum.org/docs/inter
 https://geth.ethereum.org/docs/rpc/ns-net,https://geth.ethereum.org/docs/interacting-with-geth/rpc/ns-net,,n/a
 https://geth.ethereum.org/docs/rpc/ns-personal,https://geth.ethereum.org/docs/interacting-with-geth/rpc/ns-personal,,n/a
 https://geth.ethereum.org/docs/rpc/ns-txpool,https://geth.ethereum.org/docs/interacting-with-geth/rpc/ns-txpool,,n/a
-https://geth.ethereum.org/docs/rpc/objects,https://geth.ethereum.org/docs/interacting-with-geth/rpc/objects,,n/a
+https://geth.ethereum.org/docs/rpc/objects,https://geth.ethereum.org/,page deleted,n/a
```

Comment:
We need to update the redirects as well if we're removing this page. I have PR #172 up right now right fixes a few others, @fjl let me know if you'd like me to just add this update to either of these PRs.

Reply:
Went ahead and pushed a commit that adds/updates the redirects. I had redirected to the docs pages, but I see you're using the homepage here. I'll update the redirect to match this CSV.